src/main/java/com/rakovets/course/java/core/practice/operators/Task04.java

```diff
@@ -35,6 +35,6 @@ public static void main(String[] args) {
     static float getDistance(float milliseconds) {
         //TODO
         // Код, решающий задачу пишем ниже, при этом используя параметры метода
-        return 0.0f;
+        return SOUND_SPEED*1000/3600*milliseconds;
```

Comment:
"Violation: There isn't separating white spaces for operators OR parameters
Solution: Add white spaces"